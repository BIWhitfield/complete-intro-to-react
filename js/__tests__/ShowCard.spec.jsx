import React from 'react';
import { shallow } from 'enzyme';
import ShowCard from '../ShowCard';

describe('ShowCard', () => {
  it('renders correctly', () => {
    const component = shallow(<ShowCard />);
    expect(component).toMatchSnapshot();
  });
});
