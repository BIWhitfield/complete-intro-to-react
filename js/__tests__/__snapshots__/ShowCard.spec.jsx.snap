// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ShowCard renders correctly 1`] = `
<styled.div>
  <styled.img
    alt="undefined Show Poster"
    src="/public/img/posters/undefined"
  />
  <div>
    <h3 />
    <h4>
      (
      )
    </h4>
    <p />
  </div>
</styled.div>
`;
